subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CopyPermissionsIntegrationTest.groovy

```diff
@@ -222,4 +223,34 @@ class CopyPermissionsIntegrationTest extends AbstractIntegrationSpec {
         cleanup:
         file('src/unauthorized').mode = 0777
     }
+
+    @Requires(TestPrecondition.FILE_PERMISSIONS)
+    @Issue('https://github.com/gradle/gradle/issues/9576')
+    def "unreadable excluded #type is ignored"() {
+        given:
+        def taskName = "withUnreadable$type"
```

Comment:
The task name resolves to `withUnreadablefile` etc., i.e. the last bit is all lowercase, which is weird. Instead of fixing that problem why not just call it `copyWithUnreadableOutput`? Just `copy` would also be fine given there's no other task, and the whole test is very simple.

Reply:
Done